aspnetcore/fundamentals/openapi/openapi-comments.md

```diff
@@ -0,0 +1,193 @@
+* Parameter documentation with name, description, examples.
+* Response documentation with status codes and descriptions.
+* Support for examples and deprecated markers.
+
+### `<inheritdoc/>`
+
+The generator supports [`<inheritdoc>`](/dotnet/csharp/language-reference/xmldoc/recommended-tags#inheritdoc) tags, which inherit documentation *as long as they exist in the compilation assembly*. `<inheritdoc />` tags indicate that comments must be resolved from:
+
+* Base classes
+* Implemented interfaces
+* Base methods for overrides
+
+When `<inheritdoc />` is placed on a symbol, The source generator:
+
+* Uses its knowledge of the symbol in the compilation to discover base classes and interfaces associated with the symbol.
+* Supports resolving them automatically
+* Substitutes generic type parameters in inherited documentation comments, preserving type references across inheritance boundaries.
+
+The automatic resolution behavior is currently available for XML documentation comments that exist in the assembly under compilation, and ***not*** XML documentation tags that are in referenced projects or packages. In the later scenario, XML documentation comments:
+
+* Are only presented as text.
+* Don't provide a trivial strategy for:
+  * Associating the text content to compilation symbols.
+  * Developing an understanding of the inheritance hierarchy associated with the types.
+
+XML documentation comments from `ProjectReferences` are automatically resolved and don't require additional configuration.
+
+### Supported XML documentation tags
+
+* [`<summary>`](/dotnet/csharp/language-reference/xmldoc/recommended-tags#summary) 
+* [`<remarks>`](/dotnet/csharp/language-reference/xmldoc/recommended-tags#remarks)
+* [`<param>`](/dotnet/csharp/language-reference/xmldoc/recommended-tags#param)
+* [`<returns>`](/dotnet/csharp/language-reference/xmldoc/recommended-tags#returns)
+* [`<response>`](/dotnet/csharp/language-reference/xmldoc/recommended-tags#response)
+* [`<example>`](/dotnet/csharp/language-reference/xmldoc/recommended-tags#example)
+* [`<deprecated>`](/dotnet/csharp/language-reference/xmldoc/recommended-tags#deprecated)
+* [`<inheritdoc>`](/dotnet/csharp/language-reference/xmldoc/recommended-tags#inheritdoc)
+
+### Document HTTP responses
+
+Document HTTP responses with the `<response>` tag and the `code` attribute:
+
+:::code language="csharp" source="~/fundamentals/openapi/samples/10.x/aspnet-openapi-xml/api/ProjectBoardApis.cs" id="snippet_1" highlight="6-7":::
+
+<!--```csharp
+/// <response code="200">Success response with data</response>
+/// <response code="404">Resource not found</response>
+```-->
+
+### Add examples
+
+To add examples to documentation, use the [`<example>`](/dotnet/csharp/language-reference/xmldoc/recommended-tags#example) tag for types or the `example` attribute for parameters:
+
+```csharp
+/// <example>{"name":"Sample","value":42}</example>
+/// <param name="id" example="42">The unique identifier</param>
+```
```

Comment:
I think these sections should be moved up, out of the section on the Source Generator implementation. I think a good spot would be just before the section "Customizing XML Documentation Behavior"